src/osip-simulation-view-interface/src/main/java/edu/kit/pse/osip/simulation/controller/SimulationControlInterface.java

```diff
@@ -37,4 +37,8 @@
      * Show the control window
      */
     void show();
+    /**
+     * Closes the window
+     */
+    void close();
```

Comment:
Das hatte ich als close() gemacht, um es mit dem SimulationSettingsInterface konsistent zu halten.

Alernativ in der Implementierung close() statt hide() verwenden? (Aus der API zu close():     Closes this Stage. This call is equivalent to hide().)